backpack-common/src/main/java/net/skyscanner/backpack/calendar2/CalendarState.kt

```diff
@@ -28,7 +28,7 @@ import net.skyscanner.backpack.util.InternalBackpackApi
  * @param selection the current Calendar selection
  */
 @Immutable
-data class CalendarState internal constructor(
```

Comment:
this class will eventually move to compose module once we get rid of the xml calendar version.